### ISSUE TITLE
aria-required-attr should consider fallback values

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4481]
In ARIA 1.2 the W3C removed all default values from properties that were set to be required. We updated axe-core accordingly, requiring attributes that had not been before. What we had missed is that after they did this, a table was added which re-introduced these default values as "fallback values" in a separate section of ARIA:

https://www.w3.org/TR/wai-aria-1.3/#document-handling_author-errors_states-properties

The example for which this came up was aria-level on role=heading. This should not fail axe-core. All major screen readers seem to report this as a level 2 heading:

```html
<div role=heading>Hello world</div>
```

Passing that doesn't seem like an issue. I'm much more concerned about having axe-core pass things like checkboxes without aria-checked, or a combobox without aria-controls or aria-expanded. Those will need more discussion.

**Comment by WilcoFiers** [comment]
This change is going to impact our ACT implementation. We should do something in ACT to ensure they make the same change we make here.
